tests/test_views.py

```diff
@@ -2848,25 +2848,6 @@ def _make_pointer_only_user_can_see(self, user, project, save=False):
         node = ProjectFactory(creator=user)
         project.add_pointer(node, auth=Auth(user=user), save=save)
 
-    def test_pointer_list_write_contributor_can_remove_private_component_entry(self):
```

Comment:
I spoke with product about it, and the desired functionality is to maintain current behavior in the components widget for project admins only, so that they can still remove links that no one can see. No other users should be able to see the link unless they can view the node it points to.